mule-extensions-api-persistence/src/main/java/org/mule/extension/api/persistence/PersistenceConstants.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.persistence;
+
+import org.mule.extension.api.introspection.ExtensionModel;
+import org.mule.extension.api.introspection.ModelProperty;
+import org.mule.extension.api.introspection.property.ConnectionHandlingTypeModelProperty;
+import org.mule.extension.api.introspection.property.DisplayModelProperty;
+import org.mule.extension.api.introspection.property.StudioModelProperty;
+import org.mule.extension.api.introspection.property.XmlModelProperty;
+import org.mule.extension.api.metadata.MetadataModelProperty;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+
+import java.util.Map;
+
+/**
+ * Holds constants for the serialization and desarialization of {@link ExtensionModel}s
+ *
+ * @since 1.0
+ */
+final class PersistenceConstants
```

Comment:
how are these relevant if I use binary serialization? What if we add xml serialization with a hyphenized syntax?